_implementors/servers.md

```diff
@@ -21,7 +21,7 @@ index: 1
 | C/C++/Objective-C | [Jacob Dufault](https://github.com/jacobdufault), [MaskRay](https://github.com/MaskRay), [topisani](https://github.com/topisani) | [cquery](https://github.com/jacobdufault/cquery) |
 | CSS/LESS/SASS | MS | [vscode-css-languageservice](https://github.com/Microsoft/vscode-css-languageservice) |
 | [Ceylon](https://ceylon-lang.org/)| [John Vasileff](https://github.com/jvasileff) | [vscode-ceylon](https://github.com/jvasileff/vscode-ceylon) |
-| [Clojure](https://clojure.org/)] [Case Nelson](https://github.com/snoe) | [clojure-lsp](https://github.com/snoe/clojure-lsp) |
+| [Clojure](https://clojure.org/)] | [Case Nelson](https://github.com/snoe) | [clojure-lsp](https://github.com/snoe/clojure-lsp) |

```

Comment:
Oops - the closing square bracket shouldn't be there either.

Reply:
I checked the file and it looks good to me. What do I miss?

Reply:
`[Clojure](https://clojure.org/)]` should be `[Clojure](https://clojure.org/)`

Reply:
Thanks. Fixed.